components/Article/Codebox/__tests__/index.test.tsx

```diff
@@ -42,9 +42,9 @@ describe('Codebox component (one lang)', (): void => {
 
     const buttonElement = container.querySelector('[aria-hidden=true]');
 
-    expect(buttonElement).not.toBeNull();
-
-    await user.click(buttonElement!);
+    if (buttonElement !== null) {
+      await user.click(buttonElement);
+    }
```

Comment:
the pattern you'll see here simply checks for the element. if it is not present, we don't click it, and the `expects` fail

Reply:
Button should always be present, hence instead of adding a null check, maybe adding a `!` makes sense. This should fail if the button hasn't been found. It's even ok to give an Exception as it makes it even more evident something broke...

Can we revert this then?

Reply:
yeah I was hoping to use RTL queries here but there is not user-facing content to grab onto in every case, it's just an icon IIRC.

will rework

Reply:
reverted, and added the file-level disable of again @typescript-eslint/no-non-null-assertion